functions/pipeline/taggedimages/__init__.py

```diff
@@ -35,19 +35,24 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
             # DB configuration
             data_access = ImageTagDataAccess(get_postgres_provider())
             user_id = data_access.create_user(user_name)
-            image_id_to_urls = data_access.get_tag_complete_images(image_count, user_id)
-            image_urls = list(image_id_to_urls.values())
 
-            image_id_to_image_tags = {}
-            for image_id in image_id_to_urls.keys():
-                image_id_to_image_tags[image_id] = data_access.get_image_tags(image_id)
+            # Get ready to tag images
+            ready_to_tag_images = data_access.get_ready_to_tag_images(image_count, user_id)
+
+            image_infos = data_access.get_image_info_for_image_ids(list(ready_to_tag_images.keys()))
+
+            # Get tag complete images
+            image_id_to_urls = data_access.get_tag_complete_images(user_id)
+            obj = list(image_id_to_urls.keys())
+            image_id_to_image_tags = data_access.get_image_tags_for_image_ids(obj)
 
             existing_classifications_list = data_access.get_existing_classifications()
 
             vott_json = create_starting_vott_json(image_id_to_urls, image_id_to_image_tags, existing_classifications_list)
 
-            return_body_json = {"imageUrls": image_urls,
-                                "vottJson": vott_json}
+            return_body_json = {"imageUrls": image_id_to_urls,

```

Comment:
I am overloading this one for now because it wasn't being used anywhere else, and I plan to separate it out but wanted to get something out there for us to have for testing the training pipeline. I can separate it out as part of this PR itself.